### ISSUE TITLE
Stackframe only shows Unknown Source when debugging class which is calling methods from other classes in the same project

### ISSUE THREAD

**Issue by sceee** [Microsoft/vscode-java-debug#218]
I want to debug a JUnit test class. I tried this both with Java Test Runner 0.2.0 and Debugger for Java 0.5.0 by including a main method in my test class.
Unfortunately as soon as my Test class calls a method from any other class from the same project (a normal class with some methods) and I try to jump into a method from this other class with the debugger, I only get a "unknown source" frame on top of the callstack.
It is showing me the line number and the method which is called, but I can't debug into that class (which is located in the same project, just another directory).

As long as I stay in the Test class with the debugger, I can normally step over/into methods.

I am using gradle to build.

##### Environment
- Operating System: Win10 x64
- JDK version: 1.8.0_151
- Visual Studio Code version: 1.19.2
- Java extension version: 0.2.0
- Java Debugger extension version: 0.5.0

##### Steps To Reproduce
1. Debug a test class that calls a method from another class in the project
2. Try to step into the method of the other class

##### Current Result
The topmost stack frame only has an unknown source and does I cannot debug the code in the other class.

##### Expected Result
I can directly jump into the other class to debug it.

##### Additional Informations
I already tried to include sources in the jar built by gradle but unfortunately it did not help:

```
task sourceJar(type: Jar) {
    classifier 'sources'
    from sourceSets.main.allJava
}

artifacts {
    archives sourceJar
}
```

**Comment by testforstephen** [comment]
@sceee Can you share your file structure (a screenshot is OK)? I'm trying to understand what's your case?

**Comment by sceee** [comment]
@testforstephen my file structure is basically a simple standard gradle file structure:

- ...
- gradle
- src
- - main
- - - java
- - - - de
- - - - - my
- - - - - - package
- - - - - - - MySink.java
- - - - - - - MySource.java
- test
- - java
- - - de
- - - - my
- - - - - package
- - - - - - MyTest.java
- target
- ...


So my test is in MyTest.java which basically contains a
```
@Test
public void theTest() {
     MySource ms = new MySource();
     ms.Method();
}
```

The problem is, once it tries to jump into `ms.Method()`, the debugger stays at theTest() method and the topmost stackframe is `unknown source (MySource.java #line number)`

**Comment by testforstephen** [comment]
@sceee  I tried the junit gradle project https://github.com/junit-team/junit5-samples/tree/master/junit5-vanilla-gradle, the source jump works when debugging test case. Not sure whether it's your gradle configuration issue or not. Can you try the project i listed?

**Comment by andxu** [comment]
@sceee Can you repro this issue now? We have just published new version 0.9.0 .

**Comment by sceee** [comment]
@andxu I will try to reproduce it with the new version.

**Comment by sceee** [comment]
@andxu @testforstephen sorry for the late response, it took me a while to reproduce it since it's in a not so easy to setup configuration.

Anyway, it seems to be fixed now when using Debugger for Java 0.9.0 since I can now also debug in other classes from my tests.
Thank you both!
